torchqdynamics/mesolve/mesolve.py

```diff
@@ -18,12 +18,12 @@
 
 def mesolve(
     H: TDOperatorLike,
-    jump_ops: list[OperatorLike],
+    jump_ops: OperatorLike | list[OperatorLike],
```

Comment:
I don't think we should allow a single `jump_ops` or a single `exp_ops`, it makes the object harder to understand for someone reading the code, these guys really should be lists even if they have a single element.

Reply:
(same for sesolve)

Reply:
I really like having this feature. It copies qutip behavior, it's really handy and it doesn't change much for us inside the code. I'm sure a lot of people will think this is the default behavior, so it's annoying if they get an error every time.

Reply:
Alright 👍